src/odb/test/check_3dblox.tcl

```diff
@@ -0,0 +1,80 @@
+source "helpers.tcl"
+
+proc get_3dblox_marker_count { sub_category_name } {
+  set top_chip [[ord::get_db] getChip]
+  set category [$top_chip findMarkerCategory "3DBlox"]
+  if { $category == "NULL" } {
+    return 0
+  }
+  set sub_category [$category findMarkerCategory $sub_category_name]
+  if { $sub_category == "NULL" } {
+    return 0
+  }
+  return [$sub_category getMarkerCount]
+}
+
+# Suppress noisy standard loading messages and checker warnings
+suppress_message ODB 227
+suppress_message ODB 128
+suppress_message ODB 131
+suppress_message ODB 133
+suppress_message STA 1171
+suppress_message ODB 156
+suppress_message ODB 151
```

Comment:
Is there a reason for this? usually if the logger messages are not environment/run dependent (like elapsed time or consumed memory), I prefer if they stay in the log.